assets/styles/style.css

```diff
@@ -0,0 +1,244 @@
+body {
+  margin: 0 auto;
+  font-size: 50%;
+}
+
+.header-line {
```

Comment:
Header items should properly align.
currently your items are hanging at the top.

I should see equally spaces between the image and the text both top and bottom